Primers
=========

##qPCR mRNA
To design qPCR primers with Primer3 hosted at NCBI with the following default settings, go to [my bookmark at NCBI](http://www.ncbi.nlm.nih.gov/tools/primer-blast/index.cgi?LINK_LOC=bookmark&PRIMER_PRODUCT_MIN=80&PRIMER_PRODUCT_MAX=160&PRIMER_NUM_RETURN=10&PRIMER_MIN_TM=55&PRIMER_OPT_TM=60.0&PRIMER_MAX_TM=65&PRIMER_MAX_DIFF_TM=3&PRIMER_ON_SPLICE_SITE=1&SPLICE_SITE_OVERLAP_5END=7&SPLICE_SITE_OVERLAP_3END=4&SPAN_INTRON=off&MIN_INTRON_SIZE=10&MAX_INTRON_SIZE=1000000&SEARCH_SPECIFIC_PRIMER=on&ORGANISM=Arabidopsis%20thaliana%20%28taxid%3A3702%29&PRIMER_SPECIFICITY_DATABASE=refseq_rna&TOTAL_PRIMER_SPECIFICITY_MISMATCH=1&PRIMER_3END_SPECIFICITY_MISMATCH=1&MISMATCH_REGION_LENGTH=5&PRODUCT_SIZE_DEVIATION=1000&ALLOW_TRANSCRIPT_VARIANTS=on&HITSIZE=10000&EVALUE=10000&WORD_SIZE=7&MAX_CANDIDATE_PRIMER=50000&PRIMER_MIN_SIZE=18&PRIMER_OPT_SIZE=20&PRIMER_MAX_SIZE=23&PRIMER_MIN_GC=45&PRIMER_MAX_GC=60&GC_CLAMP=2&POLYX=5&SELF_ANY=8.00&SELF_END=1&PRIMER_MISPRIMING_LIBRARY=AUTO&NO_SNP=off&LOW_COMPLEXITY_FILTER=on&MONO_CATIONS=105.7&DIVA_CATIONS=0.0&CON_ANEAL_OLIGO=50.0&CON_DNTPS=0.0&SALT_FORMULAR=1&TM_METHOD=1&PRIMER_INTERNAL_OLIGO_MIN_SIZE=18&PRIMER_INTERNAL_OLIGO_OPT_SIZE=20&PRIMER_INTERNAL_OLIGO_MAX_SIZE=27&PRIMER_INTERNAL_OLIGO_MIN_TM=57.0&PRIMER_INTERNAL_OLIGO_OPT_TM=60.0&PRIMER_INTERNAL_OLIGO_MAX_TM=63.0&PRIMER_INTERNAL_OLIGO_MAX_GC=80.0&PRIMER_INTERNAL_OLIGO_OPT_GC_PERCENT=50&PRIMER_INTERNAL_OLIGO_MIN_GC=20.0&PICK_HYB_PROBE=off&NEWWIN=off&NEWWIN=off):
- 80-160bp amplicons 
- tm 60 (salt conc for Roche Sybr Green PCR mix)
- ~20 nt oligos
- ~GC 45-60
-  3' GC clamp
- intron spanning
- blasted against the Arabidopsis genome (`Arabidopsis thaliana (taxid:3702)`)
- Database is `Refseq mRNA`
- and others...

##qPCR gDNA eg for ChIP

To design qPCR primers with Primer3 hosted at NCBI with the following default settings, go to [my bookmark at NCBI](http://www.ncbi.nlm.nih.gov/tools/primer-blast/index.cgi?LINK_LOC=bookmark&INPUT_SEQUENCE=GAAAGGCCGAAAATAAAGTTGTAAGAGATAAACCCGCCTATATAAATTCATATATTTTCCTCTCCGCTTT%0AGAATTGTCTCGTTGTCCTCCTCACTTTCATCAGCCGTTTTGAATCTCCGGCGACTTGACAGAGAAGAACA%0AAGGAAGAAGACTAAGAGAGAAAGTAAGAGATAATCCAGGAGATTCATTCTCCGTTTTGAATCTTCCTCAA%0ATCTCATCTTCTTCCGCTCTTTCTTTCCAAGGTAATAGGAACTTTCTGGATCTACTTTATTTGCTGGATCT%0ACGATCTTGTTTTCTCAATTTCCTTGAGATCTGGAATTCGTTTAATTTGGATCTGTGAACCTCCACTAAAT%0ACTTTTGGTTTTACTAGAATCGATCTAAGTTGACCGATCAGTTAGCTCGATTATAGCTACCAGAATTTGGC%0ATTGACCTTGATGGAGAGATCCATGTTCATGTTACCTGGGAAATGATTTGTATATGTGAATTGAAATCTGA%0AACTGTTGAAGTTAGATTGAATCTGAACACTGTCAATGTTAGATTGAATCTGAACACTGTTTAAGGTTAGA%0ATGAAGTTTGTGTATAGATTCTTCGAAACTTTAGGATTTGTAGTGTCGTACGTTGAACAGAAAGCTATTTC%0ATGATTCAATCAGGGTTTATTTGACTGTATTGAACTCTTTTTGTGTGTTTGCAGCTCATAAAAAATGGCTG%0AAGGCTGATGATATTCAACCAATCGTGTGTGACAATGGTACCGGTATGGTGAAGGCTGGATTTGCAGGAGA%0ATGATGCTCCCAGGGCTGTTTTTCCCAGTGTTGTTGGTAGGCCAAGACATCATGGTGTCATGGTTGGGATG%0AAACCAGAAGGATGCATATGTTGGTGATGAAGCACAATCCAAGAGAGGTATTCTTACCTTGAAGTATCCTA%0ATTGAGCATGGTGTTGTTAGCAACTGGGATGATATGGAAAAGATCTGGCATCACACTTTCTACAATGAGCT%0ATCGTATTGCTCCTGAAGAGCACCCTGTTCTTCTTACCGAGGCTCCTCTTAACCCAAAGGCCAACAGAGAG%0AAAGATGACTCAAATCATGTTTGAGACCTTTAACTCTCCCGCTATGTATGTCGCCATCCAAGCTGTTCTCT%0ACCTTGTACGCCAGTGGTCGTACAACCGGTTAGTACATTTTAGGCTTTTTGTTTATACTCATTGATCATTA%0ATTTGAACTGAGCTCTGATTATCTTCCATTGAACAGGTATTGTGCTGGATTCTGGTGATGGTGTGTCTCAC%0AACTGTGCCAATCTACGAGGGTTTCTCTCTTCCTCATGCCATCCTCCGTCTTGACCTTGCTGGACGTGACC%0ATTACTGATTACCTCATGAAGATCCTTACAGAGAGAGGTTACATGTTCACCACAACAGCAGAGCGGGAAAT%0ATGTAAGAGACATCAAGGAGAAGCTCTCCTTTGTTGCTGTTGACTACGAGCAGGAGATGGAAACCTCAAAG%0AACCAGCTCTTCCATCGAGAAGAACTATGAATTACCCGATGGGCAAGTCATCACGATTGGTGCTGAGAGAT%0ATCAGATGCCCAGAAGTCTTGTTCCAGCCCTCGTTTGTGGGAATGGAAGCTGCTGGAATCCACGAGACAAC%0ACTATAACTCAATCATGAAGTGTGATGTGGATATCAGGAAGGATCTGTACGGTAACATTGTGCTCAGTGGT%0AGGAACCACTATGTTCTCAGGTATCGCTGACCGTATGAGCAAAGAAATCACAGCACTTGCACCAAGCAGCA%0ATGAAGATTAAGGTCGTTGCACCACCTGAAAGGAAGTACAGTGTCTGGATCGGTGGTTCCATTCTTGCTTC%0ACCTCAGCACATTCCAGCAGGTAAAAATTGATCAGATTTTGTTTCGAATTCTCTTACATGCAAATTGAAAT%0ACAAAGGCTGATTATTTATGTATATGTTGGTTACAGATGTGGATCTCCAAGGCCGAGTATGATGAGGCAGG%0ATCCAGGAATCGTTCACAGAAAATGTTTCTAAGCTCTCAAGATCAAAGGCTTAAAAAGCTGGGGTTTTATG%0AAATGGGATCAAAGTTTCTTTTTTTCTTTTATATTTGCTTCTCCATTTGTTTGTTTCATTTCCCTTTTTGT%0ATTTCGTTTCTATGATGCACTTGTGTGTGACAAACTCTCTGGGTTTTTACTTACGTCTGCGTTTCAAAAAA%0AAAAAACCGCTTTCGTTTTGCGTTTTAGTCCCATTGTTTTGTAGCTCTGAGTGATCGAATTGATGCCTCTT%0ATATTCCTTTTGTTCCCTATAATTTCTTTCAAAACTCAGAAGAAAAACCTTGAAACTCTTTGCAATGTTAA%0ATATAAGTATTGTATAAGATTTTTATTGATTTGGTTATTAGTCTTACTTTTGCT&OVERLAP_5END=7&OVERLAP_3END=4&PRIMER_PRODUCT_MIN=80&PRIMER_PRODUCT_MAX=160&PRIMER_NUM_RETURN=10&PRIMER_MIN_TM=55&PRIMER_OPT_TM=60.0&PRIMER_MAX_TM=65&PRIMER_MAX_DIFF_TM=3&PRIMER_ON_SPLICE_SITE=0&SEARCHMODE=0&SPLICE_SITE_OVERLAP_5END=7&SPLICE_SITE_OVERLAP_3END=4&SPAN_INTRON=off&MIN_INTRON_SIZE=10&MAX_INTRON_SIZE=1000000&SEARCH_SPECIFIC_PRIMER=on&EXCLUDE_ENV=off&EXCLUDE_XM=off&TH_OLOGO_ALIGNMENT=off&TH_TEMPLATE_ALIGNMENT=off&ORGANISM=Arabidopsis%20thaliana%20%28taxid%3A3702%29&PRIMER_SPECIFICITY_DATABASE=refseq_representative_genomes&TOTAL_PRIMER_SPECIFICITY_MISMATCH=1&PRIMER_3END_SPECIFICITY_MISMATCH=1&MISMATCH_REGION_LENGTH=5&TOTAL_MISMATCH_IGNORE=6&MAX_TARGET_SIZE=4000&ALLOW_TRANSCRIPT_VARIANTS=on&HITSIZE=10000&EVALUE=10000&WORD_SIZE=7&MAX_CANDIDATE_PRIMER=500&PRIMER_MIN_SIZE=18&PRIMER_OPT_SIZE=20&PRIMER_MAX_SIZE=23&PRIMER_MIN_GC=45&PRIMER_MAX_GC=60&GC_CLAMP=2&NUM_TARGETS_WITH_PRIMERS=1000&NUM_TARGETS=20&MAX_TARGET_PER_TEMPLATE=100&POLYX=5&SELF_ANY=8.00&SELF_END=1&PRIMER_MAX_END_STABILITY=9&PRIMER_MAX_END_GC=5&PRIMER_MAX_TEMPLATE_MISPRIMING_TH=40.00&PRIMER_PAIR_MAX_TEMPLATE_MISPRIMING_TH=70.00&PRIMER_MAX_SELF_ANY_TH=45.0&PRIMER_MAX_SELF_END_TH=35.0&PRIMER_PAIR_MAX_COMPL_ANY_TH=45.0&PRIMER_PAIR_MAX_COMPL_END_TH=35.0&PRIMER_MAX_HAIRPIN_TH=24.0&PRIMER_MAX_TEMPLATE_MISPRIMING=12.00&PRIMER_PAIR_MAX_TEMPLATE_MISPRIMING=24.00&PRIMER_PAIR_MAX_COMPL_ANY=8.00&PRIMER_PAIR_MAX_COMPL_END=3.00&PRIMER_MISPRIMING_LIBRARY=AUTO&NO_SNP=off&LOW_COMPLEXITY_FILTER=on&MONO_CATIONS=105.7&DIVA_CATIONS=0.0&CON_ANEAL_OLIGO=50.0&CON_DNTPS=0.0&SALT_FORMULAR=1&TM_METHOD=1&PRIMER_INTERNAL_OLIGO_MIN_SIZE=18&PRIMER_INTERNAL_OLIGO_OPT_SIZE=20&PRIMER_INTERNAL_OLIGO_MAX_SIZE=27&PRIMER_INTERNAL_OLIGO_MIN_TM=57.0&PRIMER_INTERNAL_OLIGO_OPT_TM=60.0&PRIMER_INTERNAL_OLIGO_MAX_TM=63.0&PRIMER_INTERNAL_OLIGO_MAX_GC=80.0&PRIMER_INTERNAL_OLIGO_OPT_GC_PERCENT=50&PRIMER_INTERNAL_OLIGO_MIN_GC=20.0&PICK_HYB_PROBE=off&NEWWIN=off&NEWWIN=off&SHOW_SVIEWER=true):

 As above only:
 1. Imput the genomic sequence of target loci into the search box
 2. Switch of exon-exon junction requirement
 2. Change the `Database` to `Refseq Representative Genomes`
 3. When the search starts it will likely state that your sequence is *Your PCR template is highly similar to the following sequence(s) from the search database.* eg `NC_003074.8`; this is likley to be a whole chromosome sequence so do not tick that this is an intended target (tick the box).  When the search is finished there should be at least one unintended target (the whole chromosome), so just check that the coordinates on this chromosome match the location of the intended gene.
